packages/viem/src/__tests__/index-test.ts

```diff
@@ -92,11 +92,6 @@ describe("TurnkeyAccount", () => {
       signWith: expectedPrivateKeyEthAddress,
       expectedEthAddress: expectedPrivateKeyEthAddress,
     },
-    {
-      configName: "Private Key Address using createAccountWithAddress",
```

Comment:
this is a redundant config. we already test `createAccountWithAddress` using `Wallet Account using createAccountWithAddress`